home-assistant-plugin/custom_components/climate/xknx.py

```diff
@@ -128,14 +131,37 @@ def async_add_devices_config(hass, config, async_add_devices):
         group_address_on_off_state=config.get(
             CONF_ON_OFF_STATE_ADDRESS),
         override_supported_operation_modes=config.get(
-            CONF_OVERRIDE_SUPPORTED_OPERATION_MODES))
+            CONF_OVERRIDE_SUPPORTED_OPERATION_MODES),
+        override_min_temp=config.get(CONF_OVERRIDE_MIN_TEMP),
+        override_max_temp=config.get(CONF_OVERRIDE_MAX_TEMP))
     hass.data[DATA_XKNX].xknx.devices.add(climate)
     async_add_devices([KNXClimate(hass, climate)])
 
 
 class KNXClimate(ClimateDevice):
     """Representation of a KNX climate device."""
 
+    operationModeMapping = {
```

Comment:
I don't like this personally but I can't think of another way to make KNX comply with other standard climate devices out there.